web/src/ee/features/billing/server/stripeWebhookHandler.ts

```diff
@@ -618,274 +601,3 @@ async function handleSubscriptionChanged(
 
   return;
 }
-
-async function handleBillingAlertTriggered(
```

Comment:
Did you remove the corresponding events in the stripe webhook definition? If not, let me know whether I should take care of this.